src/main/java/com/powsybl/openloadflow/sensi/AcSensitivityAnalysis.java

```diff
@@ -112,6 +109,9 @@ private void calculatePostContingencySensitivityValues(List<LfSensitivityFactor<
                                                            LoadFlowParameters lfParameters, OpenLoadFlowParameters lfParametersExt,
                                                            String contingencyId, int contingencyIndex, SensitivityValueWriter valueWriter,
                                                            Reporter reporter, boolean hasTransformerBusTargetVoltage) {
+        for (LfBranch branch : lfContingency.getBranches()) {
```

Comment:
@annetill we can safely replace by LfContingency.apply ?

Reply:
It is on my TODO list, I think it should work but I need to add some tests and to perform real tests. My question is: in an AC sensitivity analysis, do you need to support transformers control such as voltage or active power control ? Do we need to support shunt voltage control ?

Reply:
For a sensitivity analysis, everything regarding control does not make really sense.